wagtail_wordpress_import/importers/import_hooks.py

```diff
@@ -33,10 +35,11 @@ def add_item_to_cache(self, hook, item):
             hook.append(item)
 
     def _get_hook_handler_data(self):
-        """Get the hook function and data to process"""
-        for hook, actions in getattr(
-            settings, "WORDPRESS_IMPORT_HOOKS_ITEMS_TO_CACHE", {}
-        ).items():
-            if hook in self.__dict__:
-                return actions["FUNCTION"], actions["DATA_TAG"]
-        return None, None
+        """Get the hook functions and data_tag names to process."""
+        return {
+            hook: (import_string(actions["FUNCTION"]), actions["DATA_TAG"])
```

Comment:
**Comment**: I moved the `import_string` step to here in this setup-esque method.